pontoon/base/models.py

```diff
@@ -479,7 +479,7 @@ def missing_strings(self):
 
     @property
     def complete(self):
-        return self.missing_strings == 0
+        return self.total_strings - self.approved_strings == 0
```

Comment:
Good call. The current version is off.

Nit: `return self.total_strings == self.approved_strings` reads more natural.

Reply:
I don't know what I was thinking :) Of course it does, thanks.